po/fr.po

```diff
@@ -0,0 +1,244 @@
+# French translations for com.github.ryonakano.spreadsheet package.
+# Copyright (C) 2019 THE com.github.ryonakano.spreadsheet'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.ryonakano.spreadsheet package.
+# NathanBnm, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.ryonakano.spreadsheet\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-06-22 16:21+0200\n"
+"PO-Revision-Date: 2019-06-22 16:00+0200\n"
+"Last-Translator: NathanBnm\n"
+"Language-Team: none\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: src/UI/MainWindow.vala:43
+msgid "Not saved yet"
+msgstr "Pas encore enregistré"
+
+#: src/UI/MainWindow.vala:138 src/UI/MainWindow.vala:401
+msgid "Spreadsheet"
+msgstr "Spreadsheet"
+
+#: src/UI/MainWindow.vala:138
+msgid "Start something new, or continue what you have been working on."
+msgstr "Commencez quelque chose de nouveau, ou continuez votre travail."
+
+#: src/UI/MainWindow.vala:139
+msgid "New Sheet"
+msgstr "Nouvelle feuille"
+
+#: src/UI/MainWindow.vala:139
+msgid "Create an empty sheet"
+msgstr "Créer une feuille vierge"
+
+#: src/UI/MainWindow.vala:140
+msgid "Open File"
+msgstr "Ouvrir un fichier"
+
+#: src/UI/MainWindow.vala:140
+msgid "Choose a saved file"
+msgstr "Sélectionner un fichier enregistré"
+
+#: src/UI/MainWindow.vala:141
+msgid "Open Last File"
+msgstr "Ouvrir le dernier fichier"
+
+#: src/UI/MainWindow.vala:141
+msgid "Continue working on foo.xlsx"
+msgstr "Continuer à travailler sur foo.xlsx"
+
+#: src/UI/MainWindow.vala:147 src/UI/MainWindow.vala:323
+#: src/UI/MainWindow.vala:461
+msgid "Open a file"
+msgstr "Ouvrir un fichier"
+
+#: src/UI/MainWindow.vala:148 src/UI/MainWindow.vala:324
+#: src/UI/MainWindow.vala:357
+msgid "_Cancel"
+msgstr "_Annuler"
+
+#: src/UI/MainWindow.vala:150 src/UI/MainWindow.vala:326
+msgid "_Open"
+msgstr "_Ouvrir"
+
+#: src/UI/MainWindow.vala:155 src/UI/MainWindow.vala:331
+#: src/UI/MainWindow.vala:364
+msgid "CSV files"
+msgstr "Fichiers CSV"
+
+#: src/UI/MainWindow.vala:184
+msgid "Insert functions to a selected cell"
+msgstr "Insérer des fonctions dans la cellule sélectionnée"
+
+#: src/UI/MainWindow.vala:186
+msgid "Click to insert numbers or functions to a selected cell"
+msgstr ""
+"Cliquez pour insérer des nombres ou des fonctions dans la cellule "
+"sélectionnée"
+
+#: src/UI/MainWindow.vala:217
+msgid "Search functions"
+msgstr "Rechercher des fonctions"
+
+#: src/UI/MainWindow.vala:244
+msgid "Set colors to letters in a selected cell"
+msgstr "Définir la couleur de police de la cellule sélectionnée"
+
+#: src/UI/MainWindow.vala:301
+#, c-format
+msgid "Untitled Spreadsheet %i"
+msgstr "Classeur sans titre %i"
+
+#: src/UI/MainWindow.vala:312
+msgid "Page 1"
+msgstr "Feuille 1"
```

Comment:
Why not "Page 1" too, to be more coherent?

Reply:
As far as I know the spreadsheet softwares uses "sheet" as a page.
"Feuille 1" is more consistent than "Page 1" in French.

Reply:
> As far as I know the spreadsheet softwares uses "sheet" as a page.

Definitely. Should this original sentence "Page 1" also update to "Sheet 1"? 🤔

Reply:
This seems more logic to me :+1:

Reply:
Created a PR: #68